Hunting Queries/Microsoft 365 Defender/General queries/Endpoint Linux AV Signature and Platform Versions

```diff
@@ -0,0 +1,23 @@
+
+name: Endpoint Linux AV Signature and Platform Versions
+description: |
+  This query will identify the Microsoft Defender Antivirus Engine version and Microsoft Defender Antivirus Security Intelligence version for Linux Servers.
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceTvmSecureConfigurationAssessment
+tactics:
+- Configuration Assessment
+- General Queries 
+- Linux
+query: |
```

Comment:
Before the Query section we also need a relevantTechniques field.
If this is not applicable the field should be left blank.